src/foam/box/network/SocketServer.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.box.network',
+  name: 'SocketServer',
+
+  implements: [
+    'foam.nanos.NanoService'
+  ],
+
+  javaImports: [
+    'foam.box.Box',
+    'foam.core.X',
+    'foam.core.ContextAware',
+    'foam.nanos.logger.Logger',
+    'java.net.ServerSocket',
+    'java.net.Socket',
+    'java.io.IOException',
+    'foam.nanos.pool.ThreadPoolAgency',
+    'foam.core.ContextAgent'
+  ],
+
+  properties: [
+    {
+      class: 'Int',
+      name: 'port',
+      value: 7070
```

Comment:
Let's use port + 2        
String portStr = System.getProperty("http.port");
Can setup in 'start'.